examples/Classification/pointcloud_classification.py

```diff
@@ -16,28 +17,43 @@
 parser.add_argument('--epochs', type=int, default=10, help='Number of train epochs.')
 parser.add_argument('-lr', '--learning-rate', type=float, default=1e-3, help='Learning rate.')
 parser.add_argument('--batch-size', type=int, default=12, help='Batch size.')
-parser.add_argument('--device', type=str, default='cuda', help='Device to use.')
+parser.add_argument('--viz-test', action='store_true', help='Visualize an output of a test sample')
+parser.add_argument('--transforms-device', type=str, default='cuda', help='Device to use.')
 
 args = parser.parse_args()
 
 
+def to_device(inp):
+    inp.to(args.transforms_device)
+    return inp
+
 transform = tfs.Compose([
+    to_device,
     tfs.TriangleMeshToPointCloud(num_samples=args.num_points),
     tfs.NormalizePointCloud()
 ])
 
+if args.transforms_device == 'cuda':
+    num_workers = 0
+    pin_memory = False
+else:
+    num_workers = cpu_count()
+    pin_memory = True
+
 train_loader = DataLoader(ModelNet(args.modelnet_root, categories=args.categories,
-                                   split='train', transform=transform, device=args.device),
-                          batch_size=args.batch_size, shuffle=True)
+                                   split='train', transform=transform),
+                          batch_size=args.batch_size, shuffle=True,
+                          num_workers=num_workers, pin_memory=pin_memory)
 
 val_loader = DataLoader(ModelNet(args.modelnet_root, categories=args.categories,
-                                 split='test', transform=transform, device=args.device),
-                        batch_size=args.batch_size)
+                                 split='test', transform=transform),
+                        batch_size=args.batch_size,
+                        num_workers=num_workers, pin_memory=pin_memory)
 
-model = PointNetClassifier(num_classes=len(args.categories)).to(args.device)
+model = PointNetClassifier(num_classes=len(args.categories)).to('cuda')
```

Comment:
We might want to run `cpu` tests too. How about adding a `device` commandline arg that specifies the device to perform training on? (Not high priority for the current PR)

Reply:
I disagree, this is not an unit test but an example, this is to show the standard usage of Kaolin, which is with GPU, I wouldn't imagine somebody starting to do DL with CPU only.

On top of that it will add big work on our CI because we will have to run that as well.

Reply:
Agree on CI load; I have a few reservations about the GPU-only advocacy, but it's cool if we currently do GPU-only.

Reply:
I'm not saying that Kaolin is GPU-only, I'm just saying that we should use CPU where it's pertinent, I may be biased but I would not not advise any user to do 3D DL with CPU-only.

Unit tests for ops should still be executed with CPU if supported.